tests/store/filters/test_status_application_version_filters.py

```diff
@@ -0,0 +1,197 @@
+from typing import List
+
+from cg.store.filters.status_application_version_filters import (
+    filter_application_versions_by_application_id,
+    filter_application_versions_by_application,
+    filter_application_versions_before_date,
+    filter_application_versions_by_version,
+)
+
+from cg.store import Store
+from cg.store.models import Application, ApplicationVersion
+from sqlalchemy import desc
+from sqlalchemy.orm import Query
+from tests.store.api.conftest import (
+    fixture_applications_store,
+    fixture_invalid_application_id,
+    fixture_invalid_application_version_version,
+)
+from tests.store_helpers import StoreHelpers
+
+
+def test_filter_application_version_by_application_correct_application(
+    store_with_applications_with_application_versions: Store,
+    helpers: StoreHelpers,
+):
+    """Test that the application of the filtered application version is the correct one."""
+    # GIVEN a store with different applications
+    applications: List[
+        Application
+    ] = store_with_applications_with_application_versions.get_applications()
+    linked_application: Application = applications[0]
+    unlinked_application: Application = applications[-1]
+    assert str(unlinked_application) != str(linked_application)
```

Comment:
is the `str()` here needed?  They should both be of type application or am I missing something?
Same for the others below.

Reply:
not really, deleting